frontend/src/pages/ActivityReport/index.css

```diff
@@ -1,13 +1,29 @@
-.smart-hub--form-section {
-  margin-top: 2rem;
+.smart-hub--form-group {
+  margin-top: 1.5rem;
```

Comment:
I think that's equal to [`.margin-105`](https://designsystem.digital.gov/utilities/margin-and-padding/) in USWDS-speak.